src/configs/ToastConfig.js

```diff
@@ -21,8 +23,8 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.black75,
     alignItems: 'center',
     justifyContent: 'center',
-    padding: 10,
-    borderRadius: 10
+    padding: dimen.normalizeDimen(10),
+    borderRadius: dimen.normalizeDimen(10)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding and border radius values have been updated to use the `normalizeDimen` function from the `dimen` module. This is a good practice as it ensures that these dimensions are scaled appropriately across different screen sizes.

```diff
-    padding: 10,
-    borderRadius: 10
+    padding: dimen.normalizeDimen(10),
+    borderRadius: dimen.normalizeDimen(10)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->